core/src/main/java/nl/nn/adapterframework/jdbc/dbms/IDbmsSupportFactory.java

```diff
@@ -30,5 +30,6 @@ public interface IDbmsSupportFactory {
 	final int DBMS_MYSQL=5;
 
 	IDbmsSupport getDbmsSupport(DataSource datasource);
+	IDbmsSupport getDbmsSupport(String product, String productVersion);
```

Comment:
Somehow I needed to change this interface, which wasn't there in 8.0 or higher

Reply:
In 8.0 and higher that interface is moved to another module, I believe? I wonder why it needs to be changed to accept strings instead of the actual dataSource, will look at the rest of the changes.

Reply:
Checking it, in 8.0 the interface was removed as it was deemed unnecessary. In 8.0 this method with 2 String parameters is called from the JdbcFacade, which has a DataSource and in 8.0 the DbmsSupportFactory class has a method to get an instance taking a Connection, and one taking these Strings.

I think in 7.9 you probably do not need to make this method public? Since the JdbcFacade has a DataSource and there's a method to get the implementation from the DataSource?

Reply:
Ok, will do that.